src/config/utils/pipeline.py

```
@@ -178,24 +180,20 @@ def run(self, input_data: np.ndarray) -> Any:
-                    self.web_interface.update_robot_position(
-                        np.zeros((4, 4), dtype=np.float32)
-                    )
                     break
                 end_time = time.time()
                 elapsed = end_time - start_time
-                if debug_mode:
-                    self.operation_time_history[i].append(elapsed)
-                    time_elapsed += elapsed
+                self.operation_time_history[i].append(elapsed)
+                time_elapsed += elapsed
             except Exception as _:
```

Ruff (0.14.0)

Do not catch blind exception: `Exception`

(BLE001)